src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -213,13 +199,97 @@ def create_new(
         return RegionResource(
             res_id=res_id,
             label=label,
-            color=color,
             region_of=region_of,
-            geometry=geometry,
+            geometry=GeometryShape(res_id),
```

Comment:
Noemi says she prefers everything to have a default value. The line width is the same default that the app uses.
As you see next, all the values can be overwritten.